java-app/pom.xml

```diff
@@ -66,7 +66,7 @@
               <goal>deploy</goal>
             </goals>
             <configuration>
-              <libertyRuntimeVersion>23.0.0.3</libertyRuntimeVersion>
+              <libertyRuntimeVersion>23.0.0.10</libertyRuntimeVersion>
```

Comment:
It seems you want to use the latest available version of Liberty runtime, if so, you should use `<libertyRuntimeVersion>[23.0.0.10,)</libertyRuntimeVersion>` which configures the minimum version and will use the newer version if it exists. Besides, `Dockerfile` also needs similar update to use the latest available Liberty runtime, for consistency.

I will push a commit later.

Reply:
Commit pushed: [6c1e9db](https://github.com/Azure-Samples/open-liberty-on-aca/pull/1/commits/6c1e9db529cb67f03b3e7406673f2ee32737862b).